libs/features/mas/web-components/build.mjs

```diff
@@ -77,6 +77,15 @@ Promise.all([
         plugins: [rewriteImports()],
         external: ['lit'],
     }),
+    build({
+      entryPoints: ['./src/cdt/countdown-timer.js'],
+      bundle: true,
+      minify: true,
```

Comment:
can you add a line below instead?
e.g: ` buildLitComponent('countdown-timer'),`

Reply:
the files for countdown-timer are placed in a separate child folder, which is not picked by  'buildLitComponent'